limit offset  param  maybe  is number 0

__Expected behavior:__
limit offset maybe  is number 0

__Actual behavior:__
when i set this offset param 0 value , throw a Exception like this:

Caused by: java.lang.IllegalArgumentException: Expecting a positive number for The number of records to skip
	at com.influxdb.Arguments.checkPositiveNumber(Arguments.java:123)
	at com.influxdb.query.dsl.functions.LimitFlux.withOffset(LimitFlux.java:81)
	at com.influxdb.query.dsl.Flux.limit(Flux.java:989)

__Specifications:__
 - Client Version: 1.12.0
 - InfluxDB Version: 2.0.0.rc0
 - JDK Version: 1.8
 - Platform: macos

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,9 @@
 ### Features
 1. [#163](https://github.com/influxdata/influxdb-client-java/pull/163): Improved logging message for retries
 
+### Bug Fixes
+1. [#161](https://github.com/influxdata/influxdb-client-java/pull/161): Offset param could be 0 - FluxDSL
+
 ## 1.12.0 [2020-10-02]
 
 ### Features

--- a/flux-dsl/src/main/java/com/influxdb/query/dsl/functions/LimitFlux.java
+++ b/flux-dsl/src/main/java/com/influxdb/query/dsl/functions/LimitFlux.java
@@ -78,7 +78,7 @@ public final class LimitFlux extends AbstractParametrizedFlux {
     @Nonnull
     public LimitFlux withOffset(final int offset) {
 
-        Arguments.checkPositiveNumber(offset, "The number of records to skip");
+        Arguments.checkNotNegativeNumber(offset, "The number of records to skip");
 
         this.withPropertyValue("offset", offset);
 

--- a/flux-dsl/src/test/java/com/influxdb/query/dsl/functions/LimitFluxTest.java
+++ b/flux-dsl/src/test/java/com/influxdb/query/dsl/functions/LimitFluxTest.java
@@ -58,6 +58,17 @@ class LimitFluxTest {
     }
 
     @Test
+    void limitOffsetZero() {
+
+        Flux flux = Flux
+                .from("telegraf")
+                .limit(100, 0);
+
+        Assertions.assertThat(flux.toString())
+                .isEqualToIgnoringWhitespace("from(bucket:\"telegraf\") |> limit(n: 100, offset:0)");
+    }
+
+    @Test
     void limitPositive() {
         Assertions.assertThatThrownBy(() -> Flux.from("telegraf").limit(-5))
                 .isInstanceOf(IllegalArgumentException.class)

